### ISSUE TITLE
Ask for full screen style support

### ISSUE THREAD

**Issue by sauntor** [elementary/applications-menu#574]
### Problem

When installed too many apps, it's hard to find my  intended app in the 5x3 grid page by page.

### Proposal

A fullscreen grid with more rows and columns may help.

1.  Grouped by the apps categories may be better, with an options to turn off this is best
2. Don't fill all space of the screen, it maybe look like `Kickerdash` (the dashboard style menu in the `KDE`)

**Comment by fayjie92** [comment]
I agree with your proposal. A fullscreen style, like the old days in elementary OS, was better for lots of installed applications.

**Comment by diogopessoa** [comment]
**Below are some suggestions**

1. Option: application menu full screen blur

![menu-blur-search-1](https://github.com/elementary/applications-menu/assets/12633864/c920d2ba-400e-42c3-8019-66c088828fc8)


2. Option: application menu full screen semi transparent blur

![Captura de tela de 2024-07-03 16-21-44](https://github.com/elementary/applications-menu/assets/12633864/12e3c0c2-3b34-418f-9e4d-3cd22c5423bf)


I made inspired by elementary's new Multitasking View style:

![photo_2024-06-06_15-27-17](https://github.com/elementary/applications-menu/assets/12633864/42b156cd-a6bc-41cf-9134-5b3eb071bc8c)

The grid style proposed in this example was 4 rows and 7 columns. But I think that in general 4 x 8 would also fit well.

The proposed _application menu grid_ makes better use of the available screen space, making it easier to find applications, as well as matching the new design style of the **Login Screen** and **Multitasking View** in elementary OS 8.

**Comment by danirabbit** [comment]
I just want to note here that we did user testing on this when we were designing the applications menu and we found that increasing the size of the grid actually makes it harder to find apps in the grid, not easier. Folks consistently found apps faster paging through more smaller grids than searching through fewer larger grids.

Full screen app launchers also don't scale well on large monitors, which is why we changed to a popover menu

**Comment by teamcons** [comment]
The category view is also very efficient when it comes to an overview of what is there or not on the system

search box is also focused by default. One or two letters are often enough to immediately find the intended app
